config/routes.rb

```diff
@@ -25,4 +25,11 @@
   resources :categories, only: [:show]
   resources :queue_items, only: [:create, :destroy]
   resources :followerships, only: [:create, :destroy]
+
+  get '/forgot_password', to: 'password_resets#new'
```

Comment:
Rails seemed to have assigned path helpers automatically, so I didn't add the `as:` option